postmaster/forms.py

```diff
@@ -5,7 +5,7 @@
 """
 
 from flask_wtf import Form
-from wtforms import StringField, PasswordField, SelectField
+from wtforms import StringField, PasswordField, SelectField, IntegerField, validators
```

Comment:
Instead of importing `validators`, import `Optional` from it on the next line. Alternatively, you can remove the line below and do `validators.DataRequired()` on line 17, 18, and 20.